projects/valtimo/config/assets/core/nl.json

```diff
@@ -480,6 +480,13 @@
       "formBuilder": "Formulier editor",
       "jsonEditor": "JSON editor",
       "output": "Uitvoer"
+    },
+    "intermediateSave": {
+      "save": "Voortgang opslaan",
+      "clear": "Invoervelden wissen",
+      "clearConfirm": "Weet u zeker dat u uw invoervelden wilt wissen? Deze actie kan niet ongedaan worden.",
```

Comment:
I think the last sentence isn't completely correct

Deze actie kan niet ongedaan worden gemaakt.